doc/stolonctl.md

```diff
@@ -1,10 +1,15 @@
 ## Stolon Client (stolonctl)
 
-`stolonctl` is the stolon client which controls the stolon cluster(s)
+`stolonctl` is the stolon client which controls the stolon cluster(s).
 
-It needs to communicate with the cluster store (providing `--store-backend` and related option) on which the requested cluster name (`--cluster-name`) is running.
+Since `stolonctl`needs to communicate with the cluster store, it requires providing the requested cluster name (`--cluster-name`), its store backend (`--store-backend`), and the kind of kubernetes resources (`--kube-resource-kind`).
 
-To avoid repeating for every command (or inside scripts) all the options you can export them as environment variables. Their name will be the same as the option name converted in uppercase, with `_` replacing `-` and prefixed with `STOLONCTL_`.
+Example of command for a cluster named "red-stolon" using "kubernetes" as a store and "configmap" as the resources:
+```
+stolonctl status --cluster-name=red-stolon --store-backend=kubernetes --kube-resource-kind=configmap
```

Comment:
This lacks the details on how the k8s api servers are contacted. This is already explained [in the architecture and requirements doc](https://github.com/sorintlab/stolon/blob/master/doc/architecture.md#kubernetes-store-backend): 

`stolonctl may be executed inside a pod running a stolon component or also externally. It'll behave like kubectl when choosing how to access the k8s API servers: When run inside a pod it'll use the pod service account to connect to the k8s API servers. When run externally it'll honor the $KUBECONFIG environment variable, use the default ~/.kube/config file or you can override the kube-config file path, the context and the namespace to use with the stolonctl options --kubeconfig, --kube-context and --kube-namespace.`

Probably a link to it could be provided or the text could be copied here.